operator/src/test/java/org/keycloak/operator/testsuite/integration/KeycloakDeploymentTest.java

```diff
@@ -498,4 +534,14 @@ public void testUpgradeRecreatesPods() {
         }
     }
 
+    private void handleFakeImagePullSecretCreation(Keycloak keycloakCR, String imagePullSecretName,
+                                                   boolean shouldMountSecretFromFile, String secretDescriptorFilename) {
+        if (shouldMountSecretFromFile) {
+            Secret imagePullSecret = getResourceFromFile(secretDescriptorFilename, Secret.class);
+            k8sclient.secrets().inNamespace(namespace).createOrReplace(imagePullSecret);
+        }
+        LocalObjectReference localObjRefAsSecretTmp = new LocalObjectReferenceBuilder().withName(imagePullSecretName).build();
```

Comment:
```suggestion
        LocalObjectReference localObjRefAsSecretTmp = new LocalObjectReferenceBuilder().withName(imagePullSecret.getMetadata().getName()).build();
```
The `imagePullSecretName ` parameter can we removed as well.

Reply:
Done! Thanks for your review and suggestion.